sdks/java/core/src/main/java/org/apache/beam/sdk/testing/BigqueryMatcher.java

```diff
@@ -121,8 +121,8 @@ protected boolean matchesSafely(PipelineResult pipelineResult) {
       throw new RuntimeException("Failed to fetch BigQuery data.", e);
```

Comment:
I believe describeMismatchSafely get called when matchesSafely successfully returns false. If we got exception here, verification will be interrupted and test fails. However, there is no documentation about the details when throw IOException but to me it may related to RPC failed.